Assets/Scripts/Controllers/TradeController.cs

```diff
@@ -104,7 +104,7 @@ private void TrasfertTradedItems(Trade trade, Vector3 tradingCoordinates)
             if (tradeItem.TradeAmount > 0)
             {
                 Tile tile = WorldController.Instance.World.GetFirstTileWithNoInventoryAround(6, (int)tradingCoordinates.x, (int)tradingCoordinates.y, (int)tradingCoordinates.z);
-                Inventory inv = new Inventory(tradeItem.Type, tradeItem.TradeAmount, tradeItem.TradeAmount);
+                Inventory inv = new Inventory(tradeItem.Type, tradeItem.TradeAmount);
```

Comment:
Than I to think of something. This will not work now at all, as maxStackSize is loaded from prototype now